CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 2022-10-19
+
+**Breaking:**
+
+- bpk-mixins: 34.0.5 => 35.0.0
+  - Deprecated `sass-functions.js` and remove `node-sass` peerDependency
```

Comment:
I would probably fall more on the side to deprecate, add a warning to the function rather than deleting the file, as that is removal rather than deprecation